src/main/java/org/opentripplanner/raptor/rangeraptor/RangeRaptorWorker.java

```diff
@@ -215,15 +205,14 @@ private void findTransitForRound() {
         var routeIndex = routeIndexIterator.next();
         var route = transitData.getRouteForIndex(routeIndex);
         var pattern = route.pattern();
-        var tripSearch = createTripSearch(route.timetable());
         var txSearch = enableTransferConstraints
           ? calculator.transferConstraintsSearch(transitData, routeIndex)
           : null;
 
         int alightSlack = slackProvider.alightSlack(pattern.slackIndex());
         int boardSlack = slackProvider.boardSlack(pattern.slackIndex());
 
-        transitWorker.prepareForTransitWith();
+        transitWorker.prepareForTransitWith(route.timetable());
```

Comment:
As the 'transitWorker' is one of the Routing strategy. Should we rename this to routingStrategy? To not confuse it with the worker and range raptor worker?

Reply:
This is not changed in this PR, I will put it on my list - there are several names that have "floated out of sync" and I will do them as part of the next PRs - which I already have started on. The strategy/worker is likely to change in the support for 1, 2 and 3 criteria routing.